src/editor/map/map_context.cpp

```diff
@@ -777,7 +777,46 @@ config map_context::to_config()
 	return scen;
 }
 
-bool map_context::save_scenario()
+void map_context::save_schedule()
+{
+	/* Create schedule config */
+	config schedule;
+	config& editor_times = schedule.add_child("editor_times");
+	config times = tod_manager_->to_config();
+	for(const config& time : times.child_range("time")) {
+		config& t = editor_times.add_child("time");
+		t.append(time);
+	}
+
+	/* Write to file */
+	try {
+		std::stringstream wml_stream;
+		wml_stream
+			<< "# This file was generated using the scenario editor.\n"
+			<< "#\n"
+			<< "#ifdef EDITOR\n";
+
+		{
+			config_writer out(wml_stream, false);
+			out.write(schedule);
+		}
+
+		wml_stream << "#endif";
+
+		if(!wml_stream.str().empty()) {
+			filesystem::write_file(filesystem::get_current_editor_dir(addon_id_) + "/utils/schedule.cfg", wml_stream.str());
+		}
+
+	} catch(const filesystem::io_exception& e) {
+		utils::string_map symbols;
+		symbols["msg"] = e.what();
+		const std::string msg = VGETTEXT("Could not save current schedule: $msg", symbols);
```

Comment:
@stevecotton this would be a new string, wouldn't it?

Reply:
Yes, it's a new string. Marking this as unresolved.

Reply:
Replaced with existing string (fcd7e8063b5e79411c1a328731a3749bc0c0399a)